### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by sreyan-ghosh** [eddiejaoude/stargate#97] [Q&A]
Hey Eddie, as I told you in the chat on YT on Access Denied, here I am to learn more and contribute to Stargate and OpenSource in general.
Could you guide me on where to **get started**?

##### In case you forgot, it's the live you did at 0630 in the morning for a bunch of college students who were in awe of your streaming set up and large coffee mug. XD

**Comment by MitanshuShaBa** [comment]
Great question. First you need to go to the issues tab and find an issue you would like to work on. The issue will have labels attached to it specifying it's priority, complexity and whether it is a bug, feature or documentation change. You can then ask to work on the issue and wait to be assigned. 
Once assigned 
- [ ] fork the project
- [ ] create a new branch to work on that issue
- [ ] once finished working create a pull request and wait for reviews from people
- [ ] work on the suggestions given during reviews

Then your pull request will either be merged or closed.

As of right now the CONTRIBUTING.md file is not ready, it contains this information. You can refer https://github.com/eddiejaoude/stargate/pull/28 till it gets merged into the main branch.

Best of luck on your open source journey 👍

**Reply by eddiejaoude** [reply]
Thank you @MitanshuShaBa for replying 👍 

@sreyan-ghosh sorry I am only getting to this now. I am slowly catching up with notifications.
